modules/grafana/local/provisioning/datasources/automatic.yml

```diff
@@ -0,0 +1,418 @@
+apiVersion: 1
+
+datasources:
```

Comment:
Our use of public-repos should avoid having anything too Mintel specific - URLs and configuration like this is Mintel specific in my view and should not be public.

I understand the reason for making use of build-harness, but perhaps configuration like this can be private?

Reply:
Some options here:

- Consider using our private repo for config and/or some extensions
- Consider passing in values (urls, password-env-references) via an env, and then generating this config (there's a lot of duplication)

Reply:
Moved this to https://gitlab.com/mintel/satoshi/tools/build-harness-extentions-private